components/TLoader.vue

```diff
@@ -1,3 +1,3 @@
 <template>
-  <div class="p-5 text-center">Loading...</div>
+  <div class="p-5 text-center">{{ $t('events.loading') }}</div>
```

Comment:
That component is used not only in the events page

```suggestion
  <div class="p-5 text-center">{{ $t('loading') }}</div>
```